package.json

```diff
@@ -8,37 +8,35 @@
   ],
   "scripts": {
     "_drop_native": "./node_modules/rimraf/bin.js node_modules/@types/react-native/",
-    "start": "npm run _drop_native && npm run testOut && npm run storybook-start",
+    "start": "yarn _drop_native && yarn testOut && yarn storybook-start",
     "storybook-start": "bash start.sh",
-    "testOut": "jest --env=jsdom --json --outputFile=.jest-test-results.json",
+    "testOut": "yarn test --env=jsdom --json --outputFile=.jest-test-results.json",
     "build-storybook": "build-storybook -c .storybook -o public",
-    "build": "npm run _drop_native && npm run testOut && build-storybook",
+    "build": "yarn _drop_native && yarn testOut && build-storybook",
     "clean": "./node_modules/rimraf/bin.js ./lib",
-    "ts-bundle": "npm run _drop_native && npm run clean && tsc && tsc -m es6 && node copy-assets",
-    "test": "jest",
-    "prepare": "install-peers && npm run ts-bundle"
+    "ts-bundle": "yarn _drop_native && yarn clean && tsc && tsc -m es6 && node copy-assets",
+    "test": "jest --config ./jest/config.js",
+    "prepare": "install-peers && yarn ts-bundle"
   },
   "keywords": [
     "uikit",
     "netdata-ui"
   ],
-  "author": "netdata frontend team",
+  "author": "netdata",
   "license": "ISC",
   "dependencies": {
     "@netdata/react-filter-box": "^1.0.0",
-    "@rmwc/button": "^5.7",
-    "@rmwc/circular-progress": "^5.7",
-    "polished": "^3.4.1",
-    "ramda": "^0.26.1",
+    "polished": "^4.0.3",
+    "ramda": "^0.27.1",
     "react-portal": "^4.2.0",
     "react-table": "^7.2.1",
-    "react-use": "^12.7.1",
+    "react-use": "^15.3.4",
```

Comment:
I think there are some breaking changes in this lib (maybe in others, too), but perhaps netdata-ui will just use different version until we update it also in Dashboard, right?

Reply:
yep. I think there are only new hooks in it tho.

Reply:
you're right, i went through the list and any breaking changes were related to hooks we don't use.